shared/django_apps/reports/migrations/0021_remove_testinstance_flaky_status_and_more.py

```diff
@@ -0,0 +1,77 @@
+# Generated by Django 4.2.11 on 2024-06-12 18:34
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+from shared.django_apps.migration_utils import RiskyAddIndex
+"""
+BEGIN;
+--
+-- Remove field flaky_status from testinstance
+--
+ALTER TABLE "reports_testinstance" DROP COLUMN "flaky_status" CASCADE;
+--
+-- Add field commits_where_fail to test
+--
+ALTER TABLE "reports_test" ADD COLUMN "commits_where_fail" text[] NULL;
+--
+-- Add field failure_rate to test
+--
+ALTER TABLE "reports_test" ADD COLUMN "failure_rate" double precision NULL;
+--
+-- Add field repoid to testinstance
+--
+ALTER TABLE "reports_testinstance" ADD COLUMN "repoid" integer NULL;
+--
+-- Create index reports_tes_commiti_b33542_idx on field(s) commitid, repoid, branch of model testinstance
+--
+CREATE INDEX "reports_tes_commiti_b33542_idx" ON "reports_testinstance" ("commitid", "repoid", "branch");
+--
+-- Create index reports_tes_repoid_9e7bb3_idx on field(s) repoid, created_at, outcome of model testinstance
+--
+CREATE INDEX "reports_tes_repoid_9e7bb3_idx" ON "reports_testinstance" ("repoid", "created_at", "outcome");
+COMMIT;
+"""
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("reports", "0020_alter_reportleveltotals_coverage_and_more"),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Is there logic using this column or has that been deleted prior to this? 

Also wouldn't this be a risky change too since it's ALTERing, therefore locking the table, and going through every row to do so?